test/EFCore.PG.FunctionalTests/Query/NetworkQueryNpgsqlTest.cs

```diff
@@ -1598,7 +1720,11 @@ public class NetContext : DbContext
             /// <param name="options">
             /// The options to be used for configuration.
             /// </param>
-            public NetContext(DbContextOptions options) : base(options) { }
+            public NetContext(DbContextOptions options) : base(options) {}
+
+            /// <inheritdoc />
+            protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+                => base.OnConfiguring(optionsBuilder.EnableSensitiveDataLogging());
```

Comment:
Out of curiosity, what is the need for this? You don't seem to actually be looking at parameter data in these tests...

Reply:
As part of the [panic](https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/issues/534#issuecomment-407144319), I was looking at the values being assigned to the parameters. Reverting this in the next commit.